### PR TITLE
Introductory principal dialogue

### PR DESCRIPTION
**Pull request by SheepTester**
This adds on to the existing introductory dialogue animation and then redirects to (the original) cocnept design, where I've snuck in a script to inject the principal (temporary image) over the screen, and he'll point out parts of the UI similar to the introduction (enter to skip, etc)

image

Idk where the smoke detectors are, so Leuf Munkler is selected temporarily instead for that part.

Dialogue is from the document. I made some minor adjustments.

### CODE REVIEW

File: cocnept-designs/v2/index.html [hunk 1]
```diff
@@ -28,15 +29,15 @@
                 background-color: #000;
                 height: 100%;
             }
-
```
Comment (SheepTester) [inline cocnept-designs/v2/index.html:31]: oops my Atom

**Review by SheepTester — COMMENTED** [review]
how do I stop a review

File: cocnept-designs/v2/index.html (hunk 1 above)
Comment (briarsychung) [reply cocnept-designs/v2/index.html:31]: 31 -

**Comment by briarsychung** [comment]
looks epic!

**Comment by SheepTester** [comment]
*Little did he know...*

**Pull request merged**